### DISCUSSION TITLE
如何改变docker  tdengine/tdengine的 默认端口

### DISCUSSION THREAD

**Discussion by wangqy1973** [taosdata/TDengine#9981] [Q&A]
我在阿里云的centos宿主机上，通过 image tdengine/tdengin (2.4.0.4)；默认端口情况下，把容器端口6030-6042/tcp/udp映射到宿主机对应端口， 运行容器后，通过我本地电脑 taos-shell，可以连接到数据库；
但是，我如果在server端配置文件 /etc/taos/taos,cfg 中修改 serverPort 为16030后； 并把容器端口16030-16042/tcp/udp映射到宿主机对应端口，重新运行容器， 然后在 我本地的电脑，通过 taos-shell ，连接 16030 端口，就不能连接到 tdengine了。 注：telnet 宿主机的 16030 端口是通的（宿主机的相应端口是通的）
请问，该如何改变docker  tdengine/tdengine的 默认端口6030-6042，如何映射到宿主机，才能在 我本地的电脑，通过 taos-shell 和 16030 端口，连接到 tdengine呢？

**Comment by yu285** [comment]
可以检查下云服务器的安全组看看是不是没有开 16030和其他衍生端口号？
